README.md

```diff
@@ -258,8 +281,7 @@ standardization of scientific software development, thus addressing
 many (if not all) core problems with scientific software development
 everyone likes to bash about (reproducibility, longevity, etc.).
```

Comment:
Also, I think section needs to expanded further on explaining the problem ( I get the gist of gist). We can add more details backing up this claim (citations or blog posts etc). If we can make this claim stronger, we convince the reader to take this up.